client/components/app.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+import axios from 'axios';
+import style from './style.js';
+import CarouselItem from './carousel.jsx';
+import LeftArrow from './leftArrow.jsx';
+import RightArrow from './rightArrow.jsx';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      images: [
+        'https://btetsy.s3.us-east-2.amazonaws.com/2_1.jpg',
+        'https://btetsy.s3.us-east-2.amazonaws.com/2_2.jpg',
+        'https://btetsy.s3.us-east-2.amazonaws.com/2_3.jpg'
+      ],
```

Comment:
Axios isn't being used. I assume not yet anyway. Maybe do a get request which gets data from the database and sets the state with the images you have in the database.